X/Xorg_xkeyboard_config/build_tarballs.jl

```diff
@@ -34,5 +35,9 @@ dependencies = [
     Dependency("Xorg_xkbcomp_jll"),
 ]
 
+init_block = raw"""
+ENV["XKB_CONFIG_ROOT"] = get(ENV, "XKB_CONFIG_ROOT", joinpath(artifact_dir, "share", "X11", "xkb"))
```

Comment:
Need `if Sys.islinux() || Sys.isfreebsd()`?  Especially now that this package became `AnyPlatform`

Reply:
OK, done!